### PR TITLE
feat: search menu toggle

### PR DESCRIPTION
**Pull request by Senna46**
Add toggle for NFT Search menu
Status display of bidding under consideration.
https://github.com/UnUniFi/lab-test/issues/87

スクリーンショット 2023-03-29 223219
スクリーンショット 2023-03-29 223231

### CODE REVIEW

File: projects/portal/src/app/views/nft-pawnshop/lenders/lenders.component.html [hunk 1]
```diff
@@ -64,25 +67,52 @@ <h2 class="card-title break-all">Bidding Status</h2>
...
     <h2 class="card-title break-all">Lend Tokens</h2>
-    <form #formRef="ngForm" (submit)="onSubmit()">
+    <div class="form-control">
+      <label class="cursor-pointer label">
+        <span class="label-text">Advanced Search</span>
+        <input
+          type="checkbox"
+          class="toggle toggle-primary"
+          name="openSearch"
+          [(ngModel)]="isOpenedSearch"
```
Comment (KimuraYu45z) [inline projects/portal/src/app/views/nft-pawnshop/lenders/lenders.component.html:77]: open search is not good word I think.
how about
"isNotFiltered"

Comment (Senna46) [reply projects/portal/src/app/views/nft-pawnshop/lenders/lenders.component.html:77]: This is only to open the input screen.

Comment (KimuraYu45z) [reply projects/portal/src/app/views/nft-pawnshop/lenders/lenders.component.html:77]: Understood but that variable naming is not good.

**Comment by Senna46** [comment]
スクリーンショット 2023-03-30 132725

**Review by KimuraYu45z — APPROVED** [review]
LGTM

**Pull request merged**